addon/webextension/selector/uicontrol.js

```diff
@@ -636,8 +636,7 @@ window.uicontrol = (function () {
         stateHandlers.resizing.startResize(event, "move");
       } else if (! ui.Box.isControl(target)) {
         mousedownPos = new Pos(event.pageX, event.pageY);
-        mouseupNoAutoselect = true;
-        setState("draggingReady");
+        setState("crosshairs");
```

Comment:
I suppose we should rename crosshairs as that mode hasn't displayed crosshairs for a long time.  But not really related to this PR.